pitivi/timeline/timeline.py

```diff
@@ -1744,6 +1744,10 @@ def __pasteClipsCb(self, unused_action, unused_parameter):
             position = self._project.pipeline.getPosition()
             copied_group_shallow_copy = self.__copied_group.paste(position)
             self.__copied_group = copied_group_shallow_copy.copy(True)
+            # Ungroup the pasted clips.
+            for child in copied_group_shallow_copy.get_children(False):
```

Comment:
I'll try later to see how it works, but .. why do we need to iterate the children of the group?

Reply:
@thiblahute do we want to fix the paste() behavior in GES instead? It would break compatibility I guess, if it matters.

Reply:
We might want to fix in GES indeed, I guess breaking the behaviour this way is fine.

@HarishFulara07 How do you feel about investigating that in GES? :-)

Reply:
Hey @aleb , that was a silly mistake on my part. A simple line of code 'copied_group_shallow_copy.ungroup(recursive=False)' would suffice. I have updated the commit :)